### PR TITLE
Get Geert's work on develop

### PR DESCRIPTION
**Pull request by richelbilderbeek**
I took a version on the `geert` branch that worked. I suggest to move it to `develop` with little review until Geert gets better at merging his work to/from develop (i.e. not wait until it is perfect) and creating junior Issues to polish things up.

### CODE REVIEW

File: R/create_map_measurements.R [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    data$lat <- as.numeric(data$lat)
+    data$lon <- as.numeric(data$lon)
+    data$car_densities <- as.numeric(data$car_densities)
+    data$car_densities <- data$car_densities / 100
+    p <- p  +
+      ggplot2::geom_point(data = data,
+                          ggplot2::aes(x = data$lon, y = data$lat),
+                          colour = I("red"), size = data$car_densities)
+  }
+  #' colour = I("red"), size = I(3), label = data$value)}
```
Comment (FrieseWoudloper) [inline R/create_map_measurements.R:25]: Comments should start with a `#`  (_not_ followed by `'`)

File: R/create_map_measurements.R [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#' Create map
+#'
+#' Fetch a map of Groningen from OpenStreetMap
+#' and save it as a PNG-file.
+#'
+#' @param png_filename name of the PNG that the map will be saved to
+#' @export
```
Comment (FrieseWoudloper) [inline R/create_map_measurements.R:7]: We should add 
`#' @importFrom magrittr "%>%"`

File: R/create_map_measurements.R [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  bbox <- osmdata::getbb("Groningen", featuretype = "state")
+  groningen <- ggmap::get_map(bbox, maptype = "toner_stamen")
+  p <- ggmap::ggmap(groningen)
+  if (show_measurements) {
+    data <- NULL; rm(data) # nolint, fixes warning: no visible  binding
+    data <- informalr::get_car_densities()
+    data$lat <- as.numeric(data$lat)
+    data$lon <- as.numeric(data$lon)
+    data$car_densities <- as.numeric(data$car_densities)
+    data$car_densities <- data$car_densities / 100
```
Comment (FrieseWoudloper) [inline R/create_map_measurements.R:19]: We could make this code more readable using pipes

**Review by FrieseWoudloper — APPROVED** [review]
Lets merge the pull request. I made some comments but we can look at them afterwards.

File: R/get_current_measurements_fast.R [hunk 4]
```diff
@@ -0,0 +1,76 @@
+#' Get current measurements of traffic
+#'
+#' @export
+#'
```
Comment (FrieseWoudloper) [inline R/get_current_measurements_fast.R:4]: We should add
`#' @importFrom magrittr "%>%"`

File: R/get_current_measurements_fast.R [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  #'value_query <-".//d1:measurementSiteName/d1:values/d1:value"
+  left <- ".//d1:measurementSiteLocation//d1:locationForDisplay/"
+  lon_q <- paste(left, "d1:longitude", sep = "")
+  lat_q <- paste(left, "d1:latitude", sep = "")
+  #'value<-xml2::xml_find_first(parents, value_query
+  #') %>% xml2::xml_text()
+  site_names_1 <- xml2::xml_find_all(doc,
+              ".//d1:measurementSiteRecord") %>% xml2::xml_attr("id")
+  lon <- xml2::xml_find_first(parents, lon_q
+                              )  %>% xml2::xml_text() #nolint
```
Comment (FrieseWoudloper) [inline R/get_current_measurements_fast.R:35]: Remove `# nolint`

**Review by janclod — COMMENTED** [review]
Let's merge :)

**Pull request merged**